.env.example

```diff
@@ -1,3 +1,3 @@
-PORT=
```

Comment:
turn back PORT and HOST vars

Reply:
i didn't delete that, only set DB_HOST for first string. That all changes!